### PR TITLE
fix: identity-provider-related links

### PR DESCRIPTION
**Pull request by max-cx**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Fixed AsciiDoc syntax in administration-guide/partials/proc_configuring-che-to-use-external-identity-provider.adoc for:
    link-identity-provider-documentation-openshift-3
    link-identity-provider-documentation-openshift-4

### What issues does this PR fix or reference?
2486

### Specify the version of the product this PR applies to.
Che 7.30.x

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/end-user-guide/nav.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 ** xref:che-theia-troubleshooting.adoc[]
 ** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[]
 * xref:workspaces-overview.adoc[]
-** xref:creating-a-workspace-from-code-sample.adoc[]
+** xref:creating-a-workspace-from-included-options.adoc[]
```
Comment (MichalMaler) [inline modules/end-user-guide/nav.adoc:13]: Hi! Please, ask the developers (or Robert) about this idea.

File: modules/end-user-guide/pages/authoring-devfiles-version-2.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [id="authoring-devfiles-version-2"]
 // 
-:navtitle: Authoring devfiles version 2
+:navtitle: Authoring a devfile 2.1.0
```
Comment (MichalMaler) [inline modules/end-user-guide/pages/authoring-devfiles-version-2.adoc:3]: Here I would go with the IBM style guide and won't be using x.y.z versions.

File: modules/end-user-guide/nav.adoc (hunk 1 above)
Comment (MichalMaler) [inline modules/end-user-guide/nav.adoc:13]: Please, don't change this.

File: modules/end-user-guide/pages/creating-a-workspace-by-importing-the-source-code-of-a-project.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project"]
-:navtitle: Creating a workspace by importing the source code of a project
+:navtitle: Creating a workspace with your project's source code
```
Comment (MichalMaler) [inline modules/end-user-guide/pages/creating-a-workspace-by-importing-the-source-code-of-a-project.adoc:2]: Keep the original. The language reviews don't include heading changes (if they are grammatically ok) .)

File: modules/end-user-guide/partials/assembly_creating-a-workspace-from-included-options.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
+:parent-context-of-creating-a-workspace-from-included-options: {context}
+
+[id="creating-a-workspace-from-included-options_{context}"]
+= Creating a workspace from included options
+
+:context: creating-a-workspace-from-included-options
+
+{prod-short} includes optional workspace samples and devfile templates to simplify workspace creation.
+
+. xref:creating-a-workspace-from-workspace-sample_{context}[Creating a workspace from workspace sample]
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/assembly_creating-a-workspace-from-included-options.adoc:10]: use * to create a bullet list of possible options. Numbered lists are for procedure steps :)

**Comment by MichalMaler** [comment]
Hello Max. Please, revert the changes of headings/names/xrefs. I spoke about this with Developers and they want to keep it that way. I will review the rest of it then. I really like the procedure steps enhancements and IBM-style application you used! Keep on roling.

File: modules/end-user-guide/partials/assembly_authoring-devfiles-version-2.adoc [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
 
 :context: authoring-devfiles-version-2
 
-This section explains the concept of the devfile 2.0 specification and how to configure a {prod-short} workspace by using a devfile 2.0. 
+Your devfile for configuring a workspace must meet the devfile 2.1.0 specification.
 
-.Prerequisites:
-* An instance of {prod-short} with the DevWorkspace Operator enabled, see xref:installation-guide:enabling-dev-workspace-engine.adoc[]
+.Prerequisites
+* An instance of {prod-short} with the xref:installation-guide:enabling-dev-workspace-engine.adoc[DevWorkspace Operator enabled]
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/assembly_authoring-devfiles-version-2.adoc:12]: DevWorkspace Operator enabled]. 
(the missing dot at the end of the sentence)

File: modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc [hunk 6]
```diff
@@ -3,49 +3,61 @@
...
+
+* xref:creating-a-custom-workspace-from-the-dashboard_{context}[Importing a project when creating a workspace]
+* xref:importing-from-the-dashboard-into-an-existing-workspace_{context}[Importing a project into a created workspace]
+* xref:importing-to-a-running-workspace-using-the-git-clone-command_{context}[Importing a project with *Git: Clone* in the GUI]
+* xref:importing-to-a-running-workspace-with-git-clone-in-a-terminal_{context}[Importing a project with `git clone` in a terminal]
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc:23]: missing dot.

File: modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc [hunk 7]
```diff
@@ -3,49 +3,61 @@
...
++
+To install {prod-short}, see the xref:installation-guide:installing-che.adoc[Installation Guide].
 
-* An existing workspace with plug-ins related to your development environment defined on this instance of {prod} xref:configuring-a-workspace-with-dashboard.adoc[].
+* A configured workspace with plug-ins related to your development environment defined on this instance of {prod} xref:configuring-a-workspace-with-dashboard.adoc[Configuring a {prod-short} {prod-ver} workspace]
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc:27]: missing dot

File: modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc [hunk 8]
```diff
@@ -107,35 +148,54 @@ image::workspaces/configure-command.png[Configure tasks]
...
-. Start a workspace, then use the *Git: Clone* command from the command palette or the Welcome screen to import a project to a running workspace.
+Import your project's codebase into a running workspace as follows:
+
+. Select the *Git: Clone* command on the *Welcome* screen or from the command palette:
+* On the *Welcome* screen, click the *Git: Clone* link.
 +
 image::workspaces/{project-context}-welcome.png[Welcome screen]
 
-. Open the command palette by using `F1`, `CTRL-SHIFT-P`, or the link displayed on the Welcome screen.
+* To open the command palette, press F1 or Ctrl+Shift+P.
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc:167]: press F1 or kbd:[Ctrl+Shift+P] (or kbd:[Cmd+Shift+p] on macOS)

File: modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc [hunk 9]
```diff
@@ -107,35 +148,54 @@ image::workspaces/configure-command.png[Configure tasks]
...
 
 [id="importing-to-a-running-workspace-with-git-clone-in-a-terminal_{context}"]
-== Importing to a running workspace with `git clone` in a terminal
+== Importing a project with `git clone` in a terminal
+
+You can use the command line to import your project into a running workspace.
 
-In addition to the approaches above, you can also start a workspace, open a *Terminal*, and type `git clone` to pull code.
+.Prerequisites
+* A running workspace
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc:182]: missing dot

File: modules/end-user-guide/partials/proc_creating-a-workspace-from-template-devfile.adoc [hunk 10]
```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+
+[id="creating-a-workspace-from-template-devfile_{context}"]
+= Creating a workspace from template devfile
+
+You can create a new workspace by using one of the devfile templates included in {prod-short}.
+
+.Prerequisites
+
+* A running instance of {prod-short}
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-from-template-devfile.adoc:10]: missing dot

File: modules/end-user-guide/partials/proc_creating-a-workspace-from-workspace-sample.adoc [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+// creating-a-workspace-from-included-options
+
+[id="creating-a-workspace-from-workspace-sample_{context}"]
+= Creating a workspace from workspace sample
+
+You can create a new workspace by using one of the workspace samples included in {prod-short}.
+
+.Prerequisites
+
+* A running instance of {prod-short}
```
Comment (MichalMaler) [inline modules/end-user-guide/partials/proc_creating-a-workspace-from-workspace-sample.adoc:12]: missing dot

**Comment by max-cx** [comment]
Created the topic branch off another topic branch by mistake.
New PR: https://github.com/eclipse/che-docs/pull/2061

**Pull request closed without merging**